packages/frontend/src/view/components/common/page/Footer.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import { L2BeatIcon } from '../icons/L2BeatIcon'
+import { StarkWareIcon } from '../icons/StarkWareIcon'
+
+export function Footer() {
+  return (
+    <footer className="flex h-16 flex-wrap items-center	justify-between gap-y-3 whitespace-normal border-t border-t-zinc-800 p-6 text-sm">
+      <span className="flex">
+        Built by <L2BeatIcon height={18} width={44} className="mx-1" /> team and
+        funded by
+        <StarkWareIcon height={18} width={98} className="ml-1" />
+      </span>
+      <span className="text-zinc-500">@Copyright 2023 L2BEAT</span>
```

Comment:
why `@Copyright` instead of just `Copyright`? also `{new Date().getFullYear()}`

Reply:
That's actually something I wanted to talk about on some meeting
Once its like this: 
![image](https://user-images.githubusercontent.com/93620601/218704605-06fcce92-8dcf-4fdf-a61d-8fc978be3d49.png)
Once like this: 
![image](https://user-images.githubusercontent.com/93620601/218704712-f2ecffd8-da4f-4ed8-842d-7777053a4bcb.png)

Reply:
Oh and now I see that it is not @ but ©.

Reply:
Let's just drop the symbol and go with `Copyright`

Reply:
sure